### PR TITLE
[DecomposedTransform] Document and implement always skewY == 0

### PR DESCRIPTION
**Pull request by behdad**
Spotted by Cosimo. I convinced myself he is right, since a and b are zero in that branch.

### CODE REVIEW

File: Lib/fontTools/misc/transform.py [hunk 1]
```diff
@@ -450,21 +462,20 @@ def fromTransform(self, transform):
...
             scaleX, scaleY = (r, delta / r)
-            skewX, skewY = (math.atan((a * c + b * d) / (r * r)), 0)
+            skewX = math.atan((a * c + b * d) / (r * r))
         elif c != 0 or d != 0:
             s = math.sqrt(c * c + d * d)
             rotation = math.pi / 2 - (
                 math.acos(-c / s) if d >= 0 else -math.acos(c / s)
             )
             scaleX, scaleY = (delta / s, s)
-            skewX, skewY = (0, math.atan((a * c + b * d) / (s * s)))
```
Comment (justvanrossum) [inline Lib/fontTools/misc/transform.py:467]: Are you saying it's never possible for this math.atan() expression to return anything but zero?

Comment (behdad) [reply Lib/fontTools/misc/transform.py:467]: Yes. No?

Comment (justvanrossum) [reply Lib/fontTools/misc/transform.py:467]: Ah, it is because we know that both a and b are zero at this point, right?

Comment (behdad) [reply Lib/fontTools/misc/transform.py:467]: Exactly.

**Review by justvanrossum — APPROVED** [review]

**Pull request merged**